contracts/tokenbridge/libraries/aeERC20.sol

```diff
@@ -27,11 +27,6 @@ import "./TransferAndCallToken.sol";
 contract aeERC20 is ERC20PermitUpgradeable, TransferAndCallToken {
     using AddressUpgradeable for address;
 
-    constructor() initializer {
```

Comment:
if this is removed, the logic contract would become initializable
while I don't think it will causes any problem in our usecase especially when self-destruct no longer delete code, it is something we should think about; it does mean that the `aeWeth` logic we deploy with CreationCodeHelper will be initializable already, which we might want to fix